docs/docs/usage/ide-support.md

```diff
@@ -1,63 +1,66 @@
-- Type information on hover
-- Go to definition (in the current project)
-- Find all references
-- Documentation on hover
-- [Worksheet mode](worksheet-mode.md)
-
-## Partially working features:
-- Completion
-- Renaming
-- Go to definition in external projects
-
-## Unimplemented features:
-- Formatting code (requires integrating with scalafmt)
-- Quick fixes (probably by integrating with scalafix)
-
-## Current limitations, to be fixed:
-- Projects should be compiled with sbt before starting the IDE, this is
-  automatically done for you if you run `sbt launchIDE`.
-- Once the IDE is started, source files that are not opened in the IDE
-  should not be modified in some other editor, the IDE won't pick up
-  these changes.
-- Not all compiler errors/warnings are displayed, just those occurring
-  during typechecking.
-
-
-Feedback
-========
-Please report issues on https://github.com/lampepfl/dotty/issues,
-you can also come chat with use on the
-[Dotty gitter channel](https://gitter.im/lampepfl/dotty)!
+IDE support for Scala 3 is available in IDEs based on [Scala Metals](https://scalameta.org/metals/)
+(e.g., Visual Studio Code, vim) and in [IntelliJ IDEA](https://www.jetbrains.com/idea/).
+
+
+## Using Visual Studio Code
+
+To use Visual Studio Code on a Scala 3 project, ensure you have the
+[Metals](https://scalameta.org/metals/docs/editors/vscode.html) plugin installed. Then open the
+project directory in VS code and click the "Import build" button in notification.
+
+
+### Under the Hood
+
+VS Code implements semantic features (such as completions, "go to definition")
+using the [Language Server Protocol (LSP)](https://github.com/Microsoft/language-server-protocol),
+so it needs a language server implementation. Metals is the implementation of LSP for Scala. It
+extracts semantic information from [semanticdb](https://scalameta.org/docs/semanticdb/guide.html),
+which is generated directly by the Scala 3 compiler.
+
+You can read more about Scala 3 support in Metals in
+[this blog post](https://medium.com/virtuslab/introduction-to-metals-with-scala-3-79ebf3120a95).
+
+To communicate with the build tool (e.g., to import the project, trigger builds, run tests),
+Metals uses the [Build Server Protocol (BSP)](https://build-server-protocol.github.io/). The
+default BSP implementation used by metals is [Bloop](https://scalacenter.github.io/bloop/), which
+supports Scala 3 projects. Alternatively,
+[sbt can be used as a BSP server](https://scalameta.org/metals/blog/2020/11/06/sbt-BSP-support.html)
+as it directly implements BSP since version 1.4.
+
+
+## Using IntelliJ IDEA
+
+IntelliJ has its own implementation for semantic features, so it does not use Metals or the
+Language Server Protocol (LSP).
+
+In order to import a project into IntelliJ there are two possibilities:
+  - Use the built-in feature to import sbt builds
+  - Use IntelliJ's support for the
+    [Build Server Protocol (BSP)](https://www.jetbrains.com/help/idea/bsp-support.html)
```

Comment:
@romanowski the bullet list is not rendered (https://dotty.epfl.ch/docs/usage/ide-support.html) - is that expected? Would be great if the rendering in scaladoc was the same as the preview on GitHub.

Reply:
It seems that this is a bug in Flexmark, empty line above is needed to render it properly. PR with fix is here: https://github.com/lampepfl/dotty/pull/11701

Reply:
Thanks! Worth a report at flexmark? I understand the situation about markdown standards and specifications is complicated, but...

https://github.com/vsch/flexmark-java#markdown-processor-emulation
> flexmark is by default commonmark compliant

https://spec.commonmark.org/0.29/#example-273
> no blank line is needed to separate a paragraph from a following list

Reply:
It needs a bit more investigation on our side to make sure that Felexmark is to blame, @BarkingBad should take a look soon